### PR TITLE
fix the CIS hardening steps when tar URL is used

### PR DESCRIPTION
**Pull request by mddamato**

### CODE REVIEW

File: roles/rke2_common/tasks/main.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@
       ((ansible_facts['os_family'] != 'RedHat' and
       ansible_facts['os_family'] != 'Rocky') or
       rke2_binary_tarball_check.stat.exists or
+      (rke2_tarball_url is defined and rke2_tarball_url != "") or
```
Comment (aleiner) [inline roles/rke2_common/tasks/main.yml:36]: You have an extra check for `rke2_tarball_url != ""` now

**Comment by aleiner** [comment]
logic for 60-rke2-cis.conf needs to be revised to support Tarball install on RHEL. check to see which exists:
/usr/share/rke2/rke2-cis-sysctl.conf
or
/usr/local/share/rke2/rke2-cis-sysctl.conf

and copy that file.

While you're at it, the reboot logic should probably be blocked to a single node at a time.

**Comment by mddamato** [comment]
I think it already does what you're asking. There is one copy 
[here](https://github.com/rancherfederal/rke2-ansible/blob/88e70d3fd0790a217cfc621fdcff11db4ea227be/roles/rke2_common/tasks/cis-hardening.yml#L20) for yum installs which comes from /usr/share/rke2/
and another
[here](https://github.com/rancherfederal/rke2-ansible/blob/88e70d3fd0790a217cfc621fdcff11db4ea227be/roles/rke2_common/tasks/cis-hardening.yml#L31) for non-yum which come from usr/local/share/rke2/

**Comment by aleiner** [comment]
right, but if you do the tarball install of a RPM system, you won't find the file.

**Review by aleiner — APPROVED** [review]

**Pull request merged**